Source/Editor/DualityEditor/DualityEditor.csproj

```diff
@@ -121,11 +121,11 @@
     <Compile Include="Extensibility\Previews\PreviewQuery.cs" />
     <Compile Include="Extensibility\Previews\IPreviewQuery.cs" />
     <Compile Include="Extensibility\Previews\PreviewSizeMode.cs" />
-    <Compile Include="Forms\ObjectRefSelectionDialog.cs">
+    <Compile Include="Forms\SelectionDialog.cs">
```

Comment:
Agree 100% to renaming the dialog, but `SelectionDialog` sounds a bit too generic. Not sure what a better name would be, but maybe something like "ObjectSelectionDialog" (without the ref) or "SearchableListSelectionDialog"? Open for suggestions.